### PR TITLE
making Poseidon even more epic

### PR DESCRIPTION
**Pull request by Strultz**
* [Fix bow sound playing twice](https://github.com/RhysB/Project-Poseidon/commit/9ff09ee3cee0d7140f0a0d013da39d0d3f0679b4)
* [Fast packets](https://github.com/RhysB/Project-Poseidon/commit/3609ce2cced8cb00af58048dfbd0c3f3f93e3ea7)
* [Option to fix the drowning pushdown](https://github.com/RhysB/Project-Poseidon/commit/f72dec19577b4263f5fde468befc68e41cb9bfd7)
* [You want spaces? here's your spaces :tf:](https://github.com/RhysB/Project-Poseidon/pull/61/commits/81ce243cda7985db8611e3a1203d7fcd4f7a91a5)
* [i hate this program!](https://github.com/RhysB/Project-Poseidon/pull/61/commits/d55704cb2d483124692f5f7540ce5d5850cb063e)
* [Shutdown and already online messages](https://github.com/RhysB/Project-Poseidon/pull/61/commits/cf33dff321b3a2bf63f9453be61b99810b8527b2)
* [the voices won't stop](https://github.com/RhysB/Project-Poseidon/pull/61/commits/fba14708fce74ea29a3af133924b386c292ca231)
* [IS THIS WHAT YOU MEANT](https://github.com/RhysB/Project-Poseidon/pull/61/commits/ad0a011da9f8f39c324647547d48ba82e7850b14)
* [:rage~1:](https://github.com/RhysB/Project-Poseidon/pull/61/commits/0e341a0bc295c8d336de0b2e826b60e7780ab7a3)

### CODE REVIEW

**Comment by CodeKid0** [comment]
so true

**Review by CodeKid0 — CHANGES_REQUESTED** [review]
Please fix formatting (too much spaces/tabs [don't use tabs])

File: src/net/minecraft/server/NetServerHandler.java [hunk 1]
```diff
@@ -133,7 +133,7 @@ public void a() {
 
     public void disconnect(String s) {
         // CraftBukkit start
-        String leaveMessage = "\u00A7e" + this.player.name + " left the game.";
+        String leaveMessage = PoseidonConfig.getInstance().getConfigString("message.player.leave").replace("%player%", this.player.name);
```
Comment (CodeKid0) [inline src/net/minecraft/server/NetServerHandler.java:136]: Could you do it the same as I did in my commits? Otherwise, everything's good!

**Review by CodeKid0 — CHANGES_REQUESTED** [review]

**Comment by CodeKid0** [comment]
btw I didn't request to remove all tabs but I'm grateful you did it.

**Comment by CodeKid0** [comment]
FIX THE TABS. FIX THE MOTHERFUCKIN TABS. THE VOICES TELL ME YOU SHOULD FIX THE TABS

**Review by CodeKid0 — APPROVED** [review]
*merge*

**Comment by RhysB** [comment]
lgtm

**Pull request merged**